Black Clovedle/ViewController.swift

```diff
@@ -6,14 +6,710 @@
 //
 
 import UIKit
+import Foundation
+import SwiftUI
+import CryptoKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, UITableViewDelegate, UITextFieldDelegate {
+    
+    @IBOutlet weak var selectedItemLabel: UILabel!
+    @IBOutlet weak var guessingTableView: UITableView!
+    @IBOutlet weak var tryAgain: UILabel!
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var guessText: UITextField!
+    @IBOutlet weak var chevron: UIButton!
+    @IBOutlet weak var submit: UIButton!
+    var tapDismissView: UIView!
```

Comment:
Outlets are fine, but for these types of views, a code formatter will probably complain about the force unwrap. I would make it optional